### DISCUSSION TITLE
Make TOTP secret hidden by default when viewing an entry via Web interface

### DISCUSSION THREAD

**Discussion by Ben0it** [dani-garcia/vaultwarden#4679] [Ideas]
To prevent shoulder surfing, protect the **TOTP secret** by making it hidden by default like a password field while showing an entry via the Web interface :

### Present state:
![image](https://github.com/dani-garcia/vaultwarden/assets/25719453/64ae948d-e309-46ca-a837-d67be9c8c5a3)

### Suggestion:
![image](https://github.com/dani-garcia/vaultwarden/assets/25719453/f8c0732c-eec5-44be-a7b7-da8fd7941377)

**Comment by BlackDex** [comment]
This is part of the clients and thus not something this project maintains. I think there already are requests for this at Bitwarden but not sure.
